versioning/scripts/dockerfiles/main.go

```diff
@@ -15,14 +15,55 @@ import (
 	"text/template"
 
 	"github.com/GoogleCloudPlatform/runtimes-common/versioning/versions"
+	"strconv"
 )
 
+type Format struct {
+	Indent string
+	Count  int
+}
+
+func parseSpecToFormat(c map[string]string) Format {
+	fIndent := " "
+	fCount := 4
+	if val, ok := c["template_retry_intendt_type"]; ok {
```

Comment:
what is `intendt_type`?

Reply:
Merged with count, now this is a user-defined intendt.